docs/resources/mv_feature_option.md

```diff
@@ -0,0 +1,59 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "flagsmith_mv_feature_option Resource - terraform-provider-flagsmith"
+subcategory: ""
+description: |-
+  Flagsmith Feature Multivariate Option
+---
+
+# flagsmith_mv_feature_option (Resource)
+
+Flagsmith Feature Multivariate Option
+
+## Example Usage
+
+```terraform
+resource "flagsmith_feature" "feature_1" {
+  feature_name = "feature_1"
+  project_uuid = "10421b1f-5f29-4da9-abe2-30f88c07c9e8"
+  description  = "This is a new multivariate feature"
+  type         = "MULTIVARIATE"
+}
+
+resource "flagsmith_mv_feature_option" "feature_1_mv_option" {
+  type                          = "unicode"
+  feature_uuid                  = flagsmith_feature.feature_1.uuid
+  string_value                  = "option_value_30"
```

Comment:
being picky now but it's a little confusing that the example has a value of `"option_value_30"` and a `default_percentage_allocation` of `60` - is this intentional?

Reply:
Ah, yes, good catch.

Reply:
done